Tests/BaseTestCase.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * Copyright 2011 Johannes M. Schmitt <schmittjoh@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace JMS\DiExtraBundle\Tests;
+
+use PHPUnit\Framework\TestCase;
+
+abstract class BaseTestCase extends TestCase
+{
+    final protected function createMock($class)
+    {
+        if (is_callable('parent::createMock')) {
```

Comment:
I'd prefer using method_exists :)

Reply:
@GuilhemN uuuuh yeah but on what should I call it? It needs an object and `$this` isn't an option https://3v4l.org/E5f2F

Reply:
`method_exists('PHPUnit\Framework\TestCase', 'createMock')` 😉

Reply:
Oh right I read too fast, `$object` can be a classname :P

Reply:
Fixed @GuilhemN :)